src/main/java/mate/academy/hibernate/relations/service/impl/AbstractService.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.hibernate.relations.service.impl;
+
+import mate.academy.hibernate.relations.service.ValidateService;
+
+public abstract class AbstractService {
+    protected final ValidateService validateService;
+
+    protected AbstractService() {
+        this.validateService = new ValidateServiceImpl();
```

Comment:
Consider using dependency injection to provide the `ValidateService` implementation, rather than instantiating it directly in the constructor. This will make the class more flexible and easier to test.